xio-tls/src/main/java/com/xjeffrose/xio/tls/X509CertificateGenerator.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
```

Comment:
pretty sure we can replace this with a light wrapper around this, maybe even just point users at this class:
https://netty.io/4.1/api/index.html?io/netty/handler/ssl/util/SelfSignedCertificate.html